builder

```diff
@@ -77,10 +77,12 @@ rsync -a "${kernel_bzimage}" "${rootfs_dir}/bzImage"
 
 # import kernel header
 mkdir -p "${rootfs_dir}/${kernel_header_dir}"
+mkdir -p "${rootfs_dir}/${kernel_header_dir}/include/arch/x86/include"
 (
   cd "${kernel_source_dir}"
   ${mk} headers_install INSTALL_HDR_PATH="${rootfs_dir}/${kernel_header_dir}" >/dev/null
   rsync -a "${kernel_source_dir}/include/" "${rootfs_dir}/${kernel_header_dir}/include/"
+  rsync -a "${kernel_source_dir}/arch/x86/include/" "${rootfs_dir}/${kernel_header_dir}/include/arch/x86/include/"
```

Comment:
Could we just rsync everything? (`${kernel_source_dir}`)

What would be the downside of that?